Tests/SwiftParserTest/translated/TryTests.swift

```diff
@@ -204,98 +188,76 @@ final class TryTests: XCTestCase {
       }
       """,
       diagnostics: [
-        // TODO: Old parser expected note on line 1: in declaration of 'TryDecl'
-        // TODO: Old parser expected error on line 2: 'try' must be placed on the initial value expression, Fix-It replacements: 3 - 7 = '', 23 - 23 = 'try '
-        // TODO: Old parser expected error on line 2: call can throw, but errors cannot be thrown out of a property initializer
-        DiagnosticSpec(locationMarker: "1️⃣", message: "unexpected text 'try' before variable"),
-        // TODO: Old parser expected error on line 3: 'try' must be placed on the initial value expression, Fix-It replacements: 3 - 7 = '', 23 - 23 = 'try '
-        // TODO: Old parser expected error on line 3: call can throw, but errors cannot be thrown out of a property initializer
-        DiagnosticSpec(locationMarker: "2️⃣", message: "unexpected text 'try' before variable"),
-        // TODO: Old parser expected error on line 4: expected declaration
-        DiagnosticSpec(locationMarker: "3️⃣", message: "unexpected text 'try' before function"),
-      ]
+        DiagnosticSpec(locationMarker: "1️⃣", message: "'try' must be placed on the initial value expression", fixIts: ["move 'try' after '='"]),
+        DiagnosticSpec(locationMarker: "2️⃣", message: "'try' must be placed on the initial value expression", fixIts: ["move 'try' after '='"]),
+        DiagnosticSpec(locationMarker: "3️⃣", message: "'try' cannot be used with 'func'"),
```

Comment:
Little bit odd, IMO a missing expression makes more sense here. If we keep this then I'd expect `try` to be unexpected and thus removed though (where as it doesn't seem to be below).

Reply:
I think what we are trying to recover from here is that the user assumed that `try` would cover the entire variable assignment and didn't recognize that `try` can only be used on expressions. So, IMHO moving the `try` makes perfect sense to me. What would your suggestion for a diagnostic + fix-its be?

Reply:
I'm only talking about (3) in this comment. (1) and (2) seem fine to me.

For 3 I'd just expect `expected expression` after the `try`. Though if it was on the same line as the `func` I'd probably expect `unexpected try, did you mean throws?` so... 🤷

Reply:
Good idea 👍